telegram-bot-simple-0.11 + telegram-bot-api-6.5 as dependencies

Build failed. Log:

```

src/Telegram/Bot/Simple/RunTG.hs:31:16: error:
    Not in scope: type constructor or class ‘AnswerWebAppQueryRequest’
    Perhaps you meant ‘AnswerInlineQueryRequest’ (imported from Telegram.Bot.API)
   |
31 | instance RunTG AnswerWebAppQueryRequest (Response SentWebAppMessage) where
   |                ^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:31:51: error:
    Not in scope: type constructor or class ‘SentWebAppMessage’
   |
31 | instance RunTG AnswerWebAppQueryRequest (Response SentWebAppMessage) where
   |                                                   ^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:35:16: error:
    Not in scope: type constructor or class ‘SendInvoiceRequest’
    Perhaps you meant one of these:
      ‘SendVoiceRequest’ (imported from Telegram.Bot.API),
      ‘SendDiceRequest’ (imported from Telegram.Bot.API)
   |
35 | instance RunTG SendInvoiceRequest (Response Message) where
   |                ^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:39:16: error:
    Not in scope: type constructor or class ‘CreateInvoiceLinkRequest’
    Perhaps you meant ‘CreateChatInviteLinkRequest’ (imported from Telegram.Bot.API)
   |
39 | instance RunTG CreateInvoiceLinkRequest (Response Text) where
   |                ^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:43:16: error:
    Not in scope:
      type constructor or class ‘AnswerShippingQueryRequest’
    Perhaps you meant ‘AnswerInlineQueryRequest’ (imported from Telegram.Bot.API)
   |
43 | instance RunTG AnswerShippingQueryRequest (Response Bool) where
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:47:16: error:
    Not in scope:
      type constructor or class ‘AnswerPreCheckoutQueryRequest’
   |
47 | instance RunTG AnswerPreCheckoutQueryRequest (Response Bool) where
   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:55:16: error:
    Not in scope: type constructor or class ‘SendGameRequest’
    Perhaps you meant one of these:
      ‘SendDiceRequest’ (imported from Telegram.Bot.API),
      ‘SendPollRequest’ (imported from Telegram.Bot.API),
      ‘SendVenueRequest’ (imported from Telegram.Bot.API)
   |
55 | instance RunTG SendGameRequest (Response Message) where
   |                ^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:59:16: error:
    Not in scope: type constructor or class ‘SetGameScoreRequest’
   |
59 | instance RunTG SetGameScoreRequest (Response SetGameScoreResult) where
   |                ^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:59:46: error:
    Not in scope: type constructor or class ‘SetGameScoreResult’
   |
59 | instance RunTG SetGameScoreRequest (Response SetGameScoreResult) where
   |                                              ^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:103:16: error:
    Not in scope: type constructor or class ‘CreateForumTopicRequest’
    |
103 | instance RunTG CreateForumTopicRequest (Response ForumTopic) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:107:16: error:
    Not in scope: type constructor or class ‘EditForumTopicRequest’
    |
107 | instance RunTG EditForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:111:16: error:
    Not in scope: type constructor or class ‘CloseForumTopicRequest’
    |
111 | instance RunTG CloseForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:115:16: error:
    Not in scope: type constructor or class ‘ReopenForumTopicRequest’
    |
115 | instance RunTG ReopenForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:119:16: error:
    Not in scope: type constructor or class ‘DeleteForumTopicRequest’
    |
119 | instance RunTG DeleteForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:123:16: error:
    Not in scope:
      type constructor or class ‘UnpinAllForumTopicMessagesRequest’
    |
123 | instance RunTG UnpinAllForumTopicMessagesRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:127:16: error:
    Not in scope:
      type constructor or class ‘EditGeneralForumTopicRequest’
    |
127 | instance RunTG EditGeneralForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:131:16: error:
    Not in scope:
      type constructor or class ‘CloseGeneralForumTopicRequest’
    |
131 | instance RunTG CloseGeneralForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:135:16: error:
    Not in scope:
      type constructor or class ‘ReopenGeneralForumTopicRequest’
    |
135 | instance RunTG ReopenGeneralForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:139:16: error:
    Not in scope:
      type constructor or class ‘HideGeneralForumTopicRequest’
    |
139 | instance RunTG HideGeneralForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/Telegram/Bot/Simple/RunTG.hs:143:16: error:
    Not in scope:
      type constructor or class ‘UnhideGeneralForumTopicRequest’
    |
143 | instance RunTG UnhideGeneralForumTopicRequest (Response Bool) where
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/telegram-bot-api/src/Telegram/Bot/API/Methods.hs
+++ b/telegram-bot-api/src/Telegram/Bot/API/Methods.hs
@@ -72,6 +72,11 @@ module Telegram.Bot.API.Methods
   , module Telegram.Bot.API.Methods.UnbanChatSenderChat
   , module Telegram.Bot.API.Methods.UnpinAllChatMessages
   , module Telegram.Bot.API.Methods.UnpinChatMessage
+
+  , module Telegram.Bot.API.Forum
+  , module Telegram.Bot.API.Games
+  , module Telegram.Bot.API.Payments
+  , module Telegram.Bot.API.WebApps
   ) where
 
 import Telegram.Bot.API.Types.ParseMode
@@ -136,3 +141,8 @@ import Telegram.Bot.API.Methods.UnbanChatMember
 import Telegram.Bot.API.Methods.UnbanChatSenderChat
 import Telegram.Bot.API.Methods.UnpinAllChatMessages
 import Telegram.Bot.API.Methods.UnpinChatMessage
+
+import Telegram.Bot.API.Forum
+import Telegram.Bot.API.Games
+import Telegram.Bot.API.Payments
+import Telegram.Bot.API.WebApps

